test/index.js

```diff
@@ -34,19 +34,19 @@ describe('eslint-config-uphold', () => {
       'mocha/no-sibling-hooks',
       'no-new',
       'new-cap',
-      'newline-before-return',
       'no-class-assign',
       'no-const-assign',
       'no-constant-condition',
       'no-dupe-class-members',
-      'no-empty',
+      'no-global-assign',
       'no-labels',
       'no-labels',
       'no-multi-str',
       'no-this-before-super',
       'no-underscore-dangle',
       'no-unused-vars',
       'padding-line-between-statements',
+      'padding-line-between-statements',
```

Comment:
Duplicated line

Reply:
That's to be expected, as with some other rules in this test, since we're testing this rule with 2 different examples.

Reply:
I see. Thanks for the clarification. 👍